scripts/ci/docker/Dockerfile

```diff
@@ -35,16 +39,26 @@ RUN apt-get update && apt-get install -y \
 
 RUN cd /usr/include && ln -s freetype2 freetype
 
-RUN groupadd -r ci && useradd -ms /bin/bash --no-log-init -r -g ci ci
-USER ci
-WORKDIR /home/ci
+#  -- Azure will add its own user, so skip this -- 
+# RUN groupadd -r ci && useradd -ms /bin/bash --no-log-init -r -g ci ci
+# USER ci
+# WORKDIR /home/ci
 
-# untar the current masonry source  (created as part of build_docker_visit_ci.sh)
-COPY visit.masonry.docker.src.tar /home/ci
+
+# untar the current masonry source  (created as part of build_docker_visit_ci.py)
+COPY visit.masonry.docker.src.tar /
 RUN tar -xzf visit.masonry.docker.src.tar
-RUN rm -rf visit.masonry.docker.src.tar
+# untar the current build_visit source (created as part of build_docker_visit_ci.py)
+RUN mkdir -p /masonry/build-mb-develop-ci-smoke/visit/src/tools/dev/scripts/
+COPY visit.build_visit.docker.src.tar /masonry
```

Comment:
Just curious by why isn't `COPY` just a `RUN cp ...`

Reply:
COPY copies a file from the host environment (say, your laptop) - into the container